lib/run/index.js

```diff
@@ -150,6 +152,11 @@ module.exports = function (options, callback) {
             (options.bail === true || (_.isObject(options.bail) && options.bail.failure))) ?
             stopOnFailure = true : stopOnFailure = false;
 
+        ignoreIteration = options.ignoreIteration;
+        if (options.iterationData) { iterationDataLength = options.iterationData.length; }
+        if (iterationDataLength && ignoreIteration > 0 && ignoreIteration < iterationDataLength) {
+            options.iterationData = options.iterationData.slice(ignoreIteration, iterationDataLength);
+        }
```

Comment:
This block seems to be adding complexity even if `options.ignoreIteration` is not set. Let's prevent this.